service/controller/resource/rbac/resource.go

```diff
@@ -0,0 +1,128 @@
+package rbac
+
+import (
+	"reflect"
+
+	"github.com/giantswarm/k8sclient/v4/pkg/k8sclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"k8s.io/api/rbac/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/prometheus-meta-operator/service/key"
+)
+
+const (
+	Name = "rbac"
+)
+
+type Config struct {
+	K8sClient k8sclient.Interface
+	Logger    micrologger.Logger
+}
+
+type Resource struct {
+	k8sClient k8sclient.Interface
+	logger    micrologger.Logger
+}
+
+func New(config Config) (*Resource, error) {
+	r := &Resource{
+		k8sClient: config.K8sClient,
+		logger:    config.Logger,
+	}
+
+	return r, nil
+}
+
+func (r *Resource) Name() string {
+	return Name
+}
+
+func toClusterRole(v interface{}) (*v1beta1.ClusterRole, error) {
+	cluster, err := key.ToCluster(v)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	name := cluster.GetName()
+
+	clusterRole := &v1beta1.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+		Rules: []v1beta1.PolicyRule{
+			{
+				APIGroups: []string{},
+				Resources: []string{
+					"nodes",
```

Comment:
I think we node node/proxy here

Reply:
What is this?

Reply:
This is so we can call the api/v1/nodes/node_name/proxy subresource IIUC

Reply:
added